pkgs/by-name/ha/hardinfo2/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  cmake,
+  pkg-config,
+  libsForQt5,
+  wrapGAppsHook4,
+
+  gtk3,
+  json-glib,
+  lerc,
+  libdatrie,
+  libepoxy,
+  libnghttp2,
+  libpsl,
+  libselinux,
+  libsepol,
+  libsoup_3,
+  libsysprof-capture,
+  libthai,
+  libxkbcommon,
+  pcre2,
+  sqlite,
+  util-linux,
+  libXdmcp,
+  libXtst,
+}:
+
+stdenv.mkDerivation {
+  pname = "hardinfo2";
+  version = "2.1.12-unstable-2024-2024-08-23";
+
+  src = fetchFromGitHub {
+    owner = "hardinfo2";
+    repo = "hardinfo2";
+    rev = "260d4c58520ec38743b8ce6554c89cf622227e90"; # features fixes for nixos
+    hash = "sha256-r29+r3KWvbdlzP3zVa/AcKT3WavanJvMtQmRDUMDVkU=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+    wrapGAppsHook4
+    libsForQt5.wrapQtAppsHook
```

Comment:
It will be double wrapped if you just include both. Do `makeWrapperArgs+=("''${qtWrapperArgs[@]}")` and disable the qt wrapper

Reply:
sorry i dont understand what you want me to do

Reply:
Add:
```
  preFixup = ''
    makeWrapperArgs+=("''${qtWrapperArgs[@]}")
  '';
  dontWrapQtApps = true;
```
Then only the other wrapper will run, but it should pick up the args of the Qt wrapper too.